CHANGES.md

```diff
@@ -1,6 +1,14 @@
 Changelog for singletons project
 ================================
 
+next
+----
+* Require GHC 8.10.
+* `singletons` now does a more much thorough job of rejecting higher-rank types
+  during promotion or singling, as they cannot support them. (Previously,
```

Comment:
I always thought that `singletons`'s preferred pronoun was "it", not "they". Regardless, using "they" and "them" in succession referring to different antecedents is confusing.